src/components/ClipboardButton/ClipboardButton.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+    Button,
+    ButtonProps,
+    ClipboardIcon,
+    CopyToClipboard as CopyToClipboardUiKit,
+    CopyToClipboardStatus,
+    Tooltip,
+} from '@gravity-ui/uikit';
+import cn from 'bem-cn-lite';
+
+const b = cn('clipboard-button');
+
+interface ClipboardButtonProps extends Pick<ButtonProps, 'disabled' | 'size' | 'title' | 'view'> {
+    className?: string;
+    text: string;
+}
+
+/**
+ * An inner component required
+ * because `react-copy-to-clipboard` doesn't work with `Tooltip` otherwise.
+ */
+function InnerButton({
+    className,
+    status,
+    title,
+    ...props
+}: Omit<ClipboardButtonProps, 'text'> & {status: CopyToClipboardStatus}) {
+    return (
+        <Tooltip content={status === CopyToClipboardStatus.Success ? 'Copied!' : title || 'Copy'}>
```

Comment:
Let's also add `placement`, probably 'right' will be the best option. Currently, when tooltip content changes, tooltip width changes as well, but not tooltip position. 

Before copy
<img width="157" alt="Screen Shot 2024-01-16 at 17 38 58" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/f283d221-d328-4c02-af8f-2c0978e9a795">

After copy
<img width="129" alt="Screen Shot 2024-01-16 at 17 38 48" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/7df430b9-79d6-4e5c-a3aa-f94c904fddfc">

Reply:
Looks like a bug in Gravity. Fixed with `placement="bottom-start"`.

I don't like the placement on right because the tooltip overlaps the current table row.